File: C++/two_sum_ii.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        unordered_map<int, int> charToInd;

        for (int i=0; i<numbers.size(); i++){
            if (charToInd.contains(target - numbers[i])){
                vector<int> res = {charToInd[target - numbers[i]], i + 1};
                return res;
            }
            charToInd[numbers[i]] = i + 1;
        }
        vector<int> res;
        return res;
    }
};

File: python/two_sum_ii.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        charToInd = {}

        for i, n in enumerate(numbers):
            if target - n in charToInd:
                return [charToInd[target - n], i + 1]
            charToInd[n] = i + 1
